src/goto-symex/symex_target_equation.cpp

```diff
@@ -389,6 +389,8 @@ void symex_target_equationt::convert_decls(
       // The result is not used, these have no impact on
       // the satisfiability of the formula.
       decision_procedure.handle(step.cond_expr);
```

Comment:
Is the handle for the `cond_expr` needed at all then?

Reply:
Yes, one of them is `ssa_lhs`, the other is `ssa_full_lhs`.